recipe/meta.yaml

```diff
@@ -20,7 +19,7 @@ build:
   - vc14  # [win and py35]
 
 # https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/issues/29
-  skip: true  # [win and py35]
+  #skip: true  # [win and py35]
```

Comment:
Probably to attempt unskipping the configuration. :) it's not obvious that you need to rerender a feedstock (since it wasn't necessary during the PR on staged-recipes either) or even that the conda-smithy tool exists.

Once you get into the conda-forge flow it all makes sense but I have to explain the whole process and reasoning behind all the tools here to colleagues and friends on a regular basis. :)